client/src/app/+admin/users/user-list/user-list.component.ts

```diff
@@ -51,6 +56,11 @@ export class UserListComponent extends RestTable implements OnInit {
         label: this.i18n('Unban'),
         handler: users => this.unbanUsers(users),
         isDisplayed: users => users.every(u => u.blocked === true)
+      },
+      {
+        label: this.i18n('Verify Email'),
+        handler: users => this.verifyUserEmails(users),
+        isDisplayed: users => this.requiresEmailVerification && users.every(u => !u.blocked && u.emailVerified === false)
```

Comment:
The reason - only those with `emailVerified === false` need to verify their email in order to login. The `emailVerified === null` case was the solution - a24cc7ed8a2dc7d789a7959b7f08ba8291ffa642 - to enable users who registered while email verification was not required to still login if email verification is enabled later.

Your question indicates that perhaps an enumeration instead of a boolean would have been a better choice for `emailVerified`. If it's worth changing now let me know and I'll work on it (as a separate issue/PR).

I'll add the requested test and respond to the other questions by Saturday at the latest.